docs/understanding-airbyte/json-avro-conversion.md

```diff
@@ -161,6 +161,149 @@ This is not supported in Avro schema. As a compromise, the converter creates a u
 }
 ```
 
+If the Json array has multiple object items, these objects will be recursively merged into one Avro record. For example, the following Json array expects two different objects, each with a different `id` field.
```

Comment:
should this doc be in undertanding airbyte? or is it more appropriately scoped for the S3/blob storage connectors docs?

Reply:
This avro doc was previously inline in the S3 and other blob storage connector docs. But as it gets longer and longer, I don't want to duplicate this doc in multiple places anymore. And In the future, more connectors using Avro / Parquet staging files will be related to this topic. So I think keeping everything here is the way to go. All relevant connector docs alreadt have links to this doc.

Reply:
I realized that the wording is slightly off. I have updated the doc in a second Avro PR: #9367.